scripts/blog-cli.js

```
@@ -20,214 +20,249 @@ const colors = {
+        case 'clear':
+          showHeader()
+          mainPrompt()
+          break
+
+        case 'status':
+        case 'series':
+        case 'upcoming':
+        case 'overdue':
+        case 'report':
+          const result = runCommand(command)
           if (result.success) {
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)